{
  DSLContext create=create(new CustomExceptionListener());
  try {
    A author=create.newRecord(TAuthor());
    author.setValue(TAuthor_ID(),1);
    author.setValue(TAuthor_LAST_NAME(),"XX");
    author.insert();
    fail();
  }
 catch (  E e) {
    assertEquals("ERROR",e.getMessage());
  }
}
